package.json

```diff
@@ -15,9 +15,9 @@
     },
     "scripts": {
         "test": "jest DFSOnBST LRU LinearSearchList BinarySearchList TwoCrystalBalls BubbleSort DoublyLinkedList Queue Stack ArrayList MazeSolver QuickSort BTPreOrder BTInOrder BTPostOrder BTBFS CompareBinaryTrees DFSOnBST DFSGraphList Trie BFSGraphMatrix Map MinHeap",
-        "clear": "./scripts/clear",
+        "clear": "node ./scripts/clear.js",
         "prettier": "prettier --write ./src",
-        "generate": "./scripts/generate",
-        "day": "echo /home/mpaulson/personal/kata-machine/src/day2"
+        "generate": "node ./scripts/generate.js",
+        "day": "echo src\\day2"
```

Comment:
if this doesn't work, it'll break me, but well willfix it.

its the \\ vs / that always worry me